### DISCUSSION TITLE
Almanac Features

### DISCUSSION THREAD

**Discussion by aphelyon-net** [mworion/MountWizzard4#234] [Feature Request] [answered]
Is it possible to compliment the moon phases, with also adding the rise, set time? We have the Upcoming twilight phases which provides enough information on rise/set of the sun. It would be a nice addition if the moon rise/set times for the users current time position had this. 
Maybe a smaller horizontal panel under the Twilight passes for: 1 month, that includes sun rise/set, moon rise/set.
Understandable if this is not possible; one can only ask.

**Comment by mworion** [comment, marked as answer]
Good point, should be not a big topic. I put it on the next release version.

**Reply by aphelyon-net** [reply]
Sorry I meant for the current day, depending on the observers location, Sun rise/set, Moon rise/set. Thanks

**Comment by mworion** [comment]
Understood, it will be like this:

![2022-04-21-14-33-22-MountWizzard4_-_v3 0 0b0](https://user-images.githubusercontent.com/6632980/164481619-2ad18b3b-036e-4872-8f1a-8186e4e57e25.png)

Please remember that "Day" in twilight definition is equal Sunrise and "Civil twilight" in the evening is equal Sunset.

Time could be either UTC or local time by setting.
Michel

**Reply by aphelyon-net** [reply]
Great work!

**Comment by aphelyon-net** [comment]
Not sure if this was added, I don't see it present in v2.2.7

**Reply by mworion** [reply]
It's part of the v3 release.
